src/core_plugins/console/api_server/es_5_0/reindex.js

```diff
@@ -0,0 +1,56 @@
+module.exports = function (api) {
+
+  api.addEndpointDescription('_post_reindex', {
+    methods: [ 'POST' ],
+    patterns: [
+      '_reindex'
+    ],
+    url_params: {
+      refresh: '__flag__',
+      wait_for_completion: 'true',
+      wait_for_active_shards: 1,
+      timeout: '30s',
+      requests_per_second: -1
```

Comment:
can we add the slices param?
https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/reindex.json#L33